src/runtime/engines/common/oci/config.go

```diff
@@ -30,6 +30,6 @@ func (c *Config) UnmarshalJSON(b []byte) error {
 	if err := json.Unmarshal(b, &c.Spec); err != nil {
 		return err
 	}
-	c.Generator = generate.NewFromSpec(&c.Spec)
+	c.Generator = generate.Generator{Config: &c.Spec}
```

Comment:
Why is this change necessary? Just curious

Reply:
Eduardo has pointed that NewFromSpec is marked as deprecated https://github.com/opencontainers/runtime-tools/blob/699dbced9a52c2d60213d8f22d6e170840d69c06/generate/generate.go#L234

Reply:
Good catch @ArangoGutierrez